example-apps/static-asset-service/nginx/Dockerfile.dev

```diff
@@ -0,0 +1,5 @@
+FROM nginx
+
+COPY index.dev.html /usr/share/nginx/html/index.html
+
+CMD sh -c "nginx && sleep 1 && echo nginx online && tail -f /dev/null"
```

Comment:
This was necessary since nginx doesn't output anything on boot up and we still rely on stdout to tell if a service is online. We will be able to remove this when we implement better ways to tell if services are online.

Reply:
Does nginx have a verbose mode?

Reply:
It does not

Reply:
Well, Heilandsack!